src/Angor/Client/Storage/ClientStorage.cs

```diff
@@ -2,45 +2,41 @@
 using Angor.Shared;
 using Angor.Shared.Models;
 using Blazored.LocalStorage;
-using Blazored.SessionStorage;
 
 namespace Angor.Client.Storage;
 
 public class ClientStorage : IClientStorage, INetworkStorage
 {
-    private readonly ISyncLocalStorageService _storage;
-    
     private const string CurrencyDisplaySettingKey = "currencyDisplaySetting";
-    
+
```

Comment:
It's very hard to know what has changed in this file, many methods have just moved around.
Can you point me to the changes in this file?